### PR TITLE
[Followup] Use asList method in some existing configOptions

### PR DESCRIPTION
**Pull request by zuston**
### What changes were proposed in this pull request?
Use asList method in some existing configOptions

### Why are the changes needed?
Directly use the asList method in ConfigOptions to get the config list values, and then avoid splitting values by users.

### Does this PR introduce _any_ user-facing change?
No.

### How was this patch tested?
UTs.

### CODE REVIEW

File: common/src/test/java/org/apache/uniffle/common/config/ConfigOptionTest.java [hunk 1]
```diff
@@ -131,6 +131,20 @@ public void testListTypes() {
...
       fail();
     }
+
+    // test the empty list
+    final ConfigOption<List<String>> emptyListStringOption = ConfigOptions
+            .key("rss.key5")
+            .stringType()
+            .asList()
+            .noDefaultValue()
+            .withDescription("List config key5");
```
Comment (jerqi) [inline common/src/test/java/org/apache/uniffle/common/config/ConfigOptionTest.java:141]: If we don't set this option and  call method conf.get(emptyListStringOption) directly, what will the result be?

Comment (zuston) [reply common/src/test/java/org/apache/uniffle/common/config/ConfigOptionTest.java:141]: It will return null.

Comment (jerqi) [reply common/src/test/java/org/apache/uniffle/common/config/ConfigOptionTest.java:141]: Is the same behavior as Flink?

Comment (zuston) [reply common/src/test/java/org/apache/uniffle/common/config/ConfigOptionTest.java:141]: Yes. I have checked.

**Review by jerqi — APPROVED** [review]
LGTM

**Pull request merged**